README.md

```diff
@@ -5,61 +5,69 @@
 [![Maven Central](https://img.shields.io/maven-central/v/io.mvnpm/locker-maven-plugin.svg?label=Maven%20Central)](https://search.maven.org/artifact/io.mvnpm/locker-maven-plugin)
 [![Apache License, Version 2.0, January 2004](https://img.shields.io/github/license/apache/maven.svg?label=License)](https://www.apache.org/licenses/LICENSE-2.0)
 
-The mvnpm locker Maven Plugin will create a version locker BOM for your `org.mvnpm` and `org.webjars` dependencies.
-Allowing you to mimick the `npm-shrinkwrap.json` and `yarn.lock` files in a Maven world.
+The mvnpm locker Maven Plugin will create a version locker profile for your `org.mvnpm` and `org.webjars` dependencies.
+Allowing you to mimick the `package-lock.json` and `yarn.lock` files in a Maven world.
 
-It is essential as NPM dependencies are typically deployed using version ranges, without locking your builds will use different versions of dependencies between builds if any of your transitive NPM based dependencies are updated. 
+_It is essential as NPM dependencies are typically deployed using version ranges, without locking your builds will use different versions of dependencies between builds if any of your transitive NPM based dependencies are updated._
```

Comment:
the cursive rendering doesn't add value does it ? just makes it harder to read?

Reply:
In my mind, it was to show that this text is just to add a bit of context. But if you think it's harder to read I can remove it